src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -134,31 +134,9 @@ public FeedbackQuestionAttributes getFeedbackQuestion(
                 fqDb.getFeedbackQuestionsForSession(feedbackSessionName, courseId);
         questions.sort(null);
 
-        if (questions.size() > 1 && !areQuestionNumbersConsistent(questions)) {
-            log.severe(courseId + ": " + feedbackSessionName + " has invalid question numbers");
-        }
-
         return questions;
     }
 
-    // TODO can be removed once we are sure that question numbers will be consistent
```

Comment:
The check is not that value-added. It dose nothing but only sending serve log to admin. Admin can also do nothing but keep receiving the logs.

The current question number adjustment logic is correct. However, under some situations (need investigation), it is inconsistent. I am thinking we should have a new issue about how to adjust question number better (because all we need is the order and order can be achieved by decimal number), so I just remove it the check (a bit radical here).

Reply:
Will add back :)